doc/decisions/decision_process.md

```diff
@@ -3,14 +3,16 @@
 ## Problem
 
 Simply discussing in an issue and then implementing a solution is okay for non-substantial changes.
-Substantial decisions must be made in a transparent and participative way.
+Substantial decisions, however, must be made in a transparent and participative way.
+
+The main purpose of the decision process is to get a common understanding of the problems and the impacts of possible solutions.
 
 ## Constraints
 
 - All relevant information about decisions must be within Elektra's repository.
 - All decisions must go through at least two review rounds, with a merge in between.
```

Comment:
I understood it as there should be two PRs for every decision. The first one introduces the problem and lists all options as considered alternatives. This will be merged after a review round. Then in a second PR one option is selected as "the decision" and this needs another review round.

Reply:
Ok, that makes sense.
Is the 2nd PR with the selected option then already the actual implementation (code) PR or another text- and discussion-focused step, e.g. defining the selected option in more detail? In the 2nd case, we'd have a 3rd review round on the actual implementation, although one can argue that this is not a part of the actual decision process.

However, I think we should explain this part of the process in more detail, e.g. which types of PRs are intended and when they should be created.

Reply:
I clarified where the steps are explained in: e3ed16ee38c7a7de287321127df12611cd738ea7

It also defines what "clear" means, and improves structure (rationale about "unbiased moved down to rationale).

> Is the 2nd PR

There are up to 4 decision PRs possible, the 2nd could, even though not recommended, be already the one that cleans up the decision to move all details to the implementation. So I cannot really answer the question as the process does not define what the 2nd PR is. Could you please rephrase the questions with the steps explained in the decision process?

Reply:
OK, so basically every step should be finished with merging a PR?
If that's the case, it should be stated in the description of the decision process.
For me it was not clear when to merge a PR and continue with a new PR and when it's recommended to push updates to an existing PR.

Reply:
Good point, this was actually not written anywhere. c28ec36f757131a2dccd34cfde05f66ca36501b0